roles/ovirt-ova-pack/files/pack_ova.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+
+import os
+import io
+import tarfile
+import sys
+import time
+
+BLOCKSIZE = 512
+NUL = "\0"
+buf = bytearray(4096)
```

Comment:
Can you please run pep8 on this file and fix all violations?

Reply:
done